testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -57,6 +60,7 @@ pub struct EmitJob {
     workers: Vec<Worker>,
     stop: Arc<AtomicBool>,
     stats: Arc<StatsAccumulator>,
+    latencies: Arc<Mutex<AtomicHistogram>>,
```

Comment:
Are you sure you need Mutex here? Should be ok without it, and Mutex kind of dangerous here as it's a high rate place

Reply:
I think we need to have a Mutex here to make consistency/atomic to read/write on this array

Reply:
But you don't write to this array (updating atomicu64 takes immutable reference for updating).
What happens if you remove mutex and use Vec?

Reply:
I think it should also works, I can remove mutex